students/Glogowska_Joanna/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,11 @@
+max = 0
+end = 0
+i = int(input('Write integers, end the sequence with 0'))
+while i != 0:
+    if i > max:
+        max = i
```

Comment:
`max` is a built-in function, you should avoid overwriting those, use `max_` or `maximum`